frontend/src/components/landing/BundleBanner.tsx

```diff
@@ -0,0 +1,182 @@
+import { FluentVariable } from "@fluent/bundle";
+import { Localized, useLocalization } from "@fluent/react";
+import {
+  getBundlePrice,
+  getBundleSubscribeLink,
+  isBundleAvailableInCountry,
+} from "../../functions/getPlan";
+import { isFlagActive } from "../../functions/waffle";
+import { RuntimeData } from "../../hooks/api/runtimeData";
+import { MozillaVpnWordmark } from "../Icons";
+import styles from "./BundleBanner.module.scss";
+import { LinkButton } from "../Button";
+import womanInBanner400w from "./images/bundle-banner-woman-400w.png";
+import womanInBanner768w from "./images/bundle-banner-woman-768w.png";
+import womanInBanner1280w from "./images/bundle-banner-woman-1280w.png";
+import bundleFloatOne from "./images/bundle-float-1.svg";
+import bundleFloatTwo from "./images/bundle-float-2.svg";
+import bundleFloatThree from "./images/bundle-float-3.svg";
+import bundleLogo from "./images/vpn-and-relay-logo.svg";
+import { trackPurchaseStart } from "../../functions/trackPurchase";
+
+export type Props = {
+  runtimeData: RuntimeData;
+};
+
+type FloatingFeaturesProps = {
+  icon: string;
+  text: string;
+  position: string;
+  vars?: Record<string, FluentVariable>;
+};
+
+const FloatingFeatures = (props: FloatingFeaturesProps) => {
+  const { l10n } = useLocalization();
+
+  const text = props.vars ? (
+    <Localized id={props.text} vars={props.vars}>
+      <span className={styles["float-features-text"]} />
+    </Localized>
+  ) : (
+    <span className={styles["float-features-text"]}>
+      {l10n.getString(props.text)}
+    </span>
+  );
+
+  return (
+    <div
+      className={`${styles[props.position]} ${styles["float-features-item"]}`}
+    >
+      <img alt="" src={props.icon} />
+      {text}
+    </div>
+  );
+};
+
+export const BundleBanner = (props: Props) => {
+  const { l10n } = useLocalization();
+
+  const mainImage = (
+    <img
+      src={womanInBanner400w.src}
+      srcSet={`${womanInBanner400w.src} 400w, ${womanInBanner768w.src} 768w, ${womanInBanner1280w.src} 1280w`}
```

Comment:
This works, which is cool, but I don't understand why! If you'd care to explain, I'd love to learn why `sizes` isn't necessary here, whereas MDN seems to say that it is required.

Reply:
Actually, maybe that's just some Firefox smarts? Chrome seems to always load the 1280w version for me, regardless of screen size.

Reply:
Hmm. I followed [this](https://crystallize.com/blog/react-srcset-for-responsive-images) tutorial, but I see what you mean about it not working.

Reply:
Changing the DPR to a lower value worked for me.

Reply:
Frontend magic ftw!